lib/hacluster.pm

```diff
@@ -1144,8 +1146,9 @@ B<$ignore_failure> do not kill the test upon failure.
 
 sub script_output_retry_check {
     my %args = @_;
-    my $cmd = $args{cmd} // die('No command specified.');
-    my $regex = $args{regex_string} // die('Regex input missing');
+    foreach (qw(cmd regex_string)) { croak "Missing mandatory $_ argument" unless $args{$_}; }
+    my $cmd = $args{cmd};
+    my $regex = $args{regex_string};
```

Comment:
I know we do this in other functions, so no issue with this change due to consistency, but in the end you replaced 2 lines, for 3. Seems less efficient.